capstone/src/main/webapp/dashboard.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="/style.css">
```

Comment:
is there a style sheet yet? if not I would recommend against referencing it until it's there!

Reply:
👍 - better to wait until this resource exists before including it.